backend/eks/kubeconfig.sh

```diff
@@ -4,12 +4,43 @@
 . ../../include/common.sh
 . .envrc
 
+
+# KUBECFG can be a kubeconfig file or a zip archive containing a tf directory for local deployments
+# KUBECFG has to be a zip archive for CI usage, see eks/deploy.sh for where a suitable file is created (tf-setup.zip)
+
 if [ ! -f "$KUBECFG" ]; then
     err "No KUBECFG given - you need to pass one!"
     exit 1
 fi
 
-cp "$KUBECFG" kubeconfig
+mtype=$(file --mime-type "$KUBECFG" |awk '{ print $2}')
+
+# Note: Current working directory is the active buildXXX environment, as needed.
+
+if [[ $mtype == "application/zip" ]] ; then
+    echo "Using Terraform state ..."
+
+    ( cd cap-terraform/eks
```

Comment:
```suggestion
    ( cd cap-terraform/eks || exit
```

Error if path doesn't exist

Reply:
Will change.

Reply:
Done with commit 70d40d388cdcbe7c471d8bdc474d7189fd976304.